Bottom: 0aec70fbd8ab98e9ea2100d0f09b4f4ec459b280
Top:    f4a4b1614f002b1220371a07a5ca3fe079865049
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-12-07 02:10:08 +0000

Refresh of refactor-polygon-contour-drawi

---

diff --git a/src/hid/common/hidgl.c b/src/hid/common/hidgl.c
index bbae50e..12f3778 100644
--- a/src/hid/common/hidgl.c
+++ b/src/hid/common/hidgl.c
@@ -621,7 +621,7 @@ fill_contour (PLINE *contour, double scale)
   /* If the contour is round, then call hidgl_fill_circle to draw it. */
   if (contour->is_round) {
     double slices = calc_slices (contour->radius / scale, 2 * M_PI);
-    if (slices < curc->Count) {
+    if (slices < contour->Count) {
       hidgl_fill_circle (contour->cx, contour->cy, contour->radius, scale);
       return;
     }
